frontend/test/metabase/scenarios/visualizations/gauge.cy.spec.js

```diff
@@ -0,0 +1,48 @@
+import { restore } from "__support__/e2e/cypress";
+import { SAMPLE_DATASET } from "__support__/e2e/cypress_sample_dataset";
+
+const { ORDERS_ID } = SAMPLE_DATASET;
+
+describe("scenarios > visualizations > gauge chart", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+    cy.intercept("POST", "/api/dataset").as("dataset");
+  });
+
+  it.skip("should not rerender on gauge arc hover (metabase#15980)", () => {
+    cy.createQuestion({
+      name: "15980",
+      query: { "source-table": ORDERS_ID, aggregation: [["count"]] },
+      display: "gauge",
+    }).then(({ body: { id: questionId } }) => {
+      cy.createDashboard("15980D").then(({ body: { id: dashboardId } }) => {
+        // Add previously created question to the dashboard
+        cy.request("POST", `/api/dashboard/${dashboardId}/cards`, {
+          cardId: questionId,
+        }).then(({ body: { id: dashCardId } }) => {
+          // Make dashboard card really small (necessary for this repro as it doesn't show any labels)
+          cy.request("PUT", `/api/dashboard/${dashboardId}/cards`, {
+            cards: [
+              {
+                id: dashCardId,
+                card_id: questionId,
+                row: 0,
+                col: 0,
+                sizeX: 6,
```

Comment:
Let's make it as small as possible.
```suggestion
                sizeX: 4,
```